### PR TITLE
Update compatibility.md

### PR DESCRIPTION
**Pull request by hugoabernier**
## Category

- [x] Content fix
- [ ] New article

## What's in this Pull Request?

Added SPFx 1.11
Changed references to release notes

### CODE REVIEW

File: docs/spfx/compatibility.md [hunk 1]
```diff
@@ -36,22 +36,22 @@ The following table lists SharePoint Framework and compatible versions of common
...
-|[1.5.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SharePoint-Framework-Package-v1.5)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.4.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4.1)|LTS 6.x,  LTS 8.x | v3, v4 |v2.4|v15.x|
-|[1.4.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.2.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-RC-Drop-1.2)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.1.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-Dev-Preview-Drop-1)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.0.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-GA)|LTS 6.x| v3 |v2.4|v15.x|
+|[1.11.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.11.0)|LTS v10.x|v5, v6|v3.3|v16.8.5|
+|[1.10.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.10.0)|LTS v10.x|v5, v6|v3.3|v16.8.5|
```
Comment (PrasadKasireddy) [inline docs/spfx/compatibility.md:40]: @hugoabernier -  Nodejs v8.0 is support with SharePoint Framework (SPFx v1.10.0) as the nodejs is using SharePoint 2016 development , the nodejs v8.0 is deprecated with SPFx v1.11.0

Comment (hugoabernier) [reply docs/spfx/compatibility.md]: Good catch! I've updated the matrix accordingly!

File: docs/spfx/compatibility.md [hunk 2]
```diff
@@ -36,22 +36,22 @@ The following table lists SharePoint Framework and compatible versions of common
...
-|[1.5.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.5.1)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.5.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SharePoint-Framework-Package-v1.5)|LTS 6.x,  LTS 8.x | v3 (w/ Node.js 6.x),<br/> v5 (w/ Node.js 8.x)|v2.4|v15.x|
-|[1.4.1](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4.1)|LTS 6.x,  LTS 8.x | v3, v4 |v2.4|v15.x|
-|[1.4.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-for-SPFx-Package-Version-1.4)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.2.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-RC-Drop-1.2)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.1.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes---Extensions-Dev-Preview-Drop-1)|LTS 6.x| v3, v4 |v2.4|v15.x|
-|[1.0.0](https://github.com/SharePoint/sp-dev-docs/wiki/Release-Notes-GA)|LTS 6.x| v3 |v2.4|v15.x|
+|[1.11.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.11.0)|LTS v10.x|v5, v6|v3.3|v16.8.5|
```
Comment (andrewconnell) [inline docs/spfx/compatibility.md:39]: Please remove the locale from the URL... let MSDocs redirect based on user agent (per MSDocs)

Comment (hugoabernier) [reply docs/spfx/compatibility.md]: Argh! Those darned locales get me every time. I've made the changes as requested

**Pull request merged**